www_som/som_indexada_webforms_helpers.py

```diff
@@ -136,5 +144,124 @@ def has_indexada_prova_pilot_category_www(self, cursor, uid, polissa_id):
             return True
         return False
 
+    def traceback_info(self, exception):
```

Comment:
Seems `traceback_info` is already defined at `www_entry_point` decorator, I think I am missing some context here, let's talk about it :speaking_head: